crates/telio-traversal/src/endpoint_providers/stun.rs

```diff
@@ -294,7 +312,8 @@ impl<Wg: WireGuard, E: Backoff> State<Wg, E> {
 
         if self
             .stun_peer_publisher
-            .try_send(self.servers.get(self.current_server_index).cloned())
+            .send(self.servers.get(self.current_server_index).cloned())
```

Comment:
Why was this change made?

Reply:
This is changed to be consistent with channel usage in other endpoint providers. I can leave `try_send`, just thought it would be nicer to do it in the same way.

Reply:
I wonder if originally `try_send()` was used instead of `send()` because there is some reason why the code can't await here if the channel buffer is full. I also wonder what are the implications if the channel won't send the message when the buffer is full.

Reply:
Yeah, because `next_server` was not `async`, so we couldn't wait here for blocking send.